service/version_bundles.go

```diff
@@ -115,5 +115,46 @@ func NewVersionBundles() []versionbundle.Bundle {
 			Version:      "1.0.0",
 			WIP:          false,
 		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "kubernetes",
+					Description: "Enable encryption at rest",
+					Kind:        "changed",
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "2.6.2",
+				},
+				{
+					Name:    "docker",
+					Version: "1.12.6",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "kubedns",
+					Version: "1.14.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.8.4",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.9.0",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "kvm-operator",
+			Time:         time.Date(2017, time.December, 19, 10, 00, 0, 0, time.UTC),
+			Version:      "2.0.0",
```

Comment:
Why is this a major bump?

Reply:
mistake, here is a `1.1.01 isn-t it?

Reply:
Feature added means minor bump to me. When it was `1.0.0` before the new one should be `1.1.0`.

Reply:
done